### PR TITLE
ref(app-platform): Upgrade echarts to latest version

### PR DESCRIPTION
**Pull request by leedongwei**
- Attempted to identify root cause of SEN-942
- Small code-style improvements on related graphing components

Refs SEN-942

### CODE REVIEW

**Review by evanpurkhiser — APPROVED** [review]
lgtm

**Comment by billyvg** [comment]
Does this fix the zoom issue?

**Review by markstory — APPROVED** [review]

**Comment by leedongwei** [comment]
@billyvg Unfortunately not. I wrote a summary in the [Jira ticket](https://getsentry.atlassian.net/browse/SEN-942).

File: src/sentry/static/sentry/app/views/events/eventsChart.jsx [hunk 1]
```diff
@@ -105,9 +105,9 @@ class EventsChart extends React.Component {
     yAxis: 'event_count',
   };
 
-  handleYAxisChange = value => {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/events/eventsChart.jsx]: I think we need this bound here

Comment (leedongwei) [reply src/sentry/static/sentry/app/views/events/eventsChart.jsx]: @billyvg I've re-checked all the bounds that I've removed and made changes in the latest commit.

**Review by dashed — APPROVED** [review]
Looks good to me 📈

**Pull request merged**